Revise Eye Condition Causes

quazi-h opened the pull request:
## Why
DAP requested a column name change hs_condition_cause to hs_eye_condition_cause because this is the only health condition for which a cause is asked with multi-select options. DAP also requested that we pull hs_dx_eye_cause for dogs with blindness to populate this variable and use a default value of 99 when no cause is provided
[Relevant ticket](https://broadinstitute.atlassian.net/browse/DSPDC-1593)

## This PR
- Renamed TDR variable hs_condition_cause to hs_eye_condition_cause.
- Updated if...else logic for blindness to inject a default value of 99 when a cause is not provided
- Updated unit tests to account for these changes

quazi-h committed:
Renamed TDR variable hs_condition_cause to hs_eye_condition_cause.
For dogs with Blindness as a health condition, we should default the TDR variable hs_eye_condition_cause to 99 when a cause is not provided.
Updated unit tests to account for this change in logic.

aherbst-broad reviewed (approved):
Looks good; is there a verification plan for this (i.e,. an example record we can verify against)?

ThePletch reviewed (approved):
lgtm, echoing drew's question about independent validation

quazi-h committed:
Merge branch 'master' into QH-DSPDC-1593-eye-condition-cause

quazi-h committed:
Merge branch 'master' into QH-DSPDC-1593-eye-condition-cause

quazi-h committed:
Added hles_health_condition tsv to the validation tests.
Added a test to check for the renamed column 'hs_eye_condition_cause'.
Added a test to check a specific dog_id for a condition.

quazi-h committed:
Replaced the test case with one where the cause for blindness was not provided in RedCap.
